bundles/framework/divmanazer/component/visualization-form/DotForm.js

```diff
@@ -22,7 +22,7 @@ Oskari.clazz.define(
 
         this.values = {
             size: this.defaultValues.size || 1,
-            color: this.defaultValues.color || 'ffde00',
+            color: '#'+this.defaultValues.color || 'ffde00',
```

Comment:
This should be  '#' + (this.defaultValues.color || 'ffde00') to work as expected?

Reply:
Yes, good point.